packages/web-worker/src/unhandled.js

```diff
@@ -0,0 +1,88 @@
+const ERROR_NAME = 'worker onerror'
+
+/*
+ * Automatically captures unhandled worker errors
+ */
+module.exports = {
```

Comment:
Should be able to use the window onerror plugin by passing `self` to it when importing it in the notifier

Reply:
Do you mean we can get rid of all the copy-pasted code below?  I think that would be good as some of the comments are strange in the context of a worker.